lib/membrane_h264_ffmpeg/parser.ex

```diff
@@ -391,12 +413,31 @@ defmodule Membrane.H264.FFmpeg.Parser do
     }
   end
 
-  defp carries_parameters_in_band?(<<payload::binary-size(40), _rest::binary>>) do
+  # Checks is the required parameter NALus (#{inspect}) are present in-band, before any video frames appears
```

Comment:
What did you want to inspect here? It's a comment, so it won't work. Did you mean to write this?
```suggestion
  # Checks if the required parameter NALus (see @required_parameter_nalus) are present in-band before any video frames appear
```